README.md

```diff
@@ -1,141 +1,101 @@
-<p align="center"><img src="https://user-images.githubusercontent.com/65415371/124739522-d8393d00-df11-11eb-9741-820955887d21.png" width="1000px" /></p>
-<!-- ALL-CONTRIBUTORS-BADGE:START - Do not remove or modify this section -->
-<a href='https://posthog.com/contributors'><img src='https://img.shields.io/badge/all_contributors-190-orange.svg?style=flat-square' /></a>
-<!-- ALL-CONTRIBUTORS-BADGE:END -->
+<p align="center">
+  <img width="300" alt="posthoglogo" src="https://user-images.githubusercontent.com/85295485/144591858-cb39e04a-7bf4-4a4d-a1fd-5d05a3ff693f.png">
+</p>
+<p align="center">
+  <!-- ALL-CONTRIBUTORS-BADGE:START - Do not remove or modify this section -->
+  <a href='https://posthog.com/contributors'><img alt="Github All Contributors" src="https://img.shields.io/github/all-contributors/posthog/posthog"></a>
+  <!-- ALL-CONTRIBUTORS-BADGE:END -->
+  <a href='http://makeapullrequest.com'><img alt='PRs Welcome' src='https://img.shields.io/badge/PRs-welcome-brightgreen.svg?style=shields'/></a>
+  <a href='https://posthog.com/slack'><img alt="Join Slack Community" src="https://img.shields.io/badge/slack%20community-join-blue"/></a>
+  <img alt="Docker Pulls" src="https://img.shields.io/docker/pulls/posthog/posthog"/>
+  <img alt="GitHub commit activity" src="https://img.shields.io/github/commit-activity/m/posthog/posthog"/>
+  <img alt="GitHub closed issues" src="https://img.shields.io/github/issues-closed/posthog/posthog"/>
+</p>
+
+## PostHog is an open-source product analytics suite, built for engineers
```

Comment:
I'm a bit worried that putting "built for engineers" in a headline may put off not-engineers – or even engineers who want a solution that just does the job, instead of one requiring actual engineering skills (`posthog.capture()` notwithstanding)

Reply:
'Built for builders'? Or 'Made for product builders'?

Reply:
This was a only one word different from the current first sentence

> PostHog is an open-source product analytics suite, built for developers

Does "developers" sound better @Twixes @charlescook-ph ? If so I can change it back.

**Motivation for the change**

Since this is the readme I'd expect the audience to be quite heavily skewed to engineers, so thought it would be nice to see something that might resonate with them, in the title.

> instead of one requiring actual engineering skills

Ah, when I wrote it I read this as the total opposite (i.e. built for engineers means it's built with my [as an engineer] needs in mind and we've made it easier  to set up [e.g. auto-capture])

Perhaps:

> PostHog is an open-source product analytics suite, built with engineers in mind

But this feels a lot weaker.

Reply:
You're right, I take it back - for a GitHub audience, I don't think putting off non-engineers should be a concern for us vs. the benefit of trying to resonate with them more.

Reply:
It's a one-word change, but I think more important is that instead of being slightly buried in the description, now this is the primary headline of the README. I kind of like "built with engineers in mind", but the reasoning makes sense, should be alright as-is